lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -293,6 +293,12 @@ def services
       end
     end
 
+    def services_payment_partial_name
+      if @services_data[ceremony_country].present?
+        @services_data[ceremony_country]['payment_partial_name']
+      end
+    end
```

Comment:
> Minor: Thinking about it a bit more, rather than use `Hash#dig`, perhaps we could extract a method like this to avoid the if statement:

@floehopper: I've extracted such a method in https://github.com/alphagov/smart-answers/pull/2507/commits/411edd96d148365e748a32ddcdead160a4d8ad16.